### ISSUE TITLE
Rebuild DEB to prevent hijacked code

### ISSUE THREAD

**Issue by mpember** [SickChill/readynas-sickchill#6]
Is there any chance of the ReadyNAS builds being updated to avoid them being 'upgraded' to the sickrage.ca code?

**Comment by Mhynlo** [comment]
I will get a build to use today.

**Comment by Mhynlo** [comment]
I created a pull request with the updates and posted the image here:
https://github.com/Mhynlo/readynas-sickchill/releases/tag/201810.15-ml0

**Comment by mpember** [comment]
Thanks. I'll try install it in the next few days

**Comment by mpember** [comment]
Sorry for the delay. Installed OK.  Only issue was that the restore process required restarting the app and not just restarting SickChill from within the menu in the SickChill web interface.

**Comment by theskillwithin** [comment]
@mpember how do u do that?

**Comment by mpember** [comment]
@theskillwithin I simply downloaded the deb and followed the usual app install process.  Prior to uninstalling the previous SickRage app, I backed up the data.  Once SickChill was installed, I restored the backup.  As mentioned, the only 'issue' was that when the restore process requested that I restart SickChill, it required that I restart the app, not just the SickChill service.

**Comment by theskillwithin** [comment]
@mpember how do you restart the app?

**Comment by mpember** [comment]
@theskillwithin you turn the app off and on again.

**Comment by theskillwithin** [comment]
@mpember I think I messed things up by installing sickchill before uninstalling sickrage.  

in attempt to get one of them working again I tried reinstalling sickrage so that I can do a backup

```
➜  Documents sudo dpkg -i sickrage_2018.02.17-ml0_all.deb
(Reading database ... 57531 files and directories currently installed.)
Preparing to unpack sickrage_2018.02.17-ml0_all.deb ...
Removed /etc/systemd/system/multi-user.target.wants/fvapp-sickrage.service.
Unpacking sickrage (2018.02.17-ml0) over (2018.02.17-ml0) ...
Setting up sickrage (2018.02.17-ml0) ...
dpkg: error: failed to write status database record about 'libtcl8.5:armel' to '/var/lib/dpkg/status': No space left on device
```

**Comment by mpember** [comment]
@theskillwithin I can't offer further assistance.  If you are doing this via an SSH session, that is beyond anything I needed to do.  I did the following via a web browser:

1. Back up SickRage data
2. Uninstall SickRage app
3. Download DEB file for SickChill
4. Install SickChill app
5. Restore SickRage data to SickChill
6. Stop SickChill app (via ReadyNAS UI, NOT SickChill UI)
7. Start SickChill app (via ReadyNAS UI)

Everything was restored and up-to-date.

If it matters, I had previously changed the GIT url in the SickRage config to point to the new SickChill repo.  Since I had only done this to avoid getting the hijacked SickRage update, that may or may not have been important.
